src/sentry/api/endpoints/project_app_store_connect_credentials.py

```diff
@@ -38,22 +38,18 @@
    and can be used in e.g. endpoint 6 and 7.  See
    :class:`AppStoreConnectCreateCredentialsEndpoint`.
 
-6. ``POST projects/{org_slug}/{proj_slug}/appstoreconnect/{id}/``
-
-   Update a subset of the full credentials normally saved in endpoint 5.  Like endpoint 5 it
```

Comment:
So I think this is being skipped because its done by step 5?

Reply:
I'm still using this endpoint when updating an app store connect config

Reply:
this has been put back